packages/regenerator-runtime/package.json

```diff
@@ -10,6 +10,7 @@
     "generator",
     "async"
   ],
+  "sideEffects": true,
```

Comment:
This is to keep Webpack from removing `import … from "regenerator-runtime"`, since the package now has an important side-effect.